inc/thorvg.h

```diff
@@ -1062,29 +1097,32 @@ class TVG_EXPORT GlCanvas final : public Canvas
  *
  * @ingroup ThorVG
  *
- * @brief description...
+ * @brief A class that enables initialization and termination of the ThorVG engine. 
  *
  */
 class TVG_EXPORT Initializer final
 {
 public:
     /**
-     * @brief ...
-     *
-     * @param[in] arg ...
-     *
-     * @note ...
-     *
-     * @return ...
-     *
-     * @see ...
+     * @brief Initializes the ThorVG engine.
+     * @param[in] engine The engine type - the same as specified in the 'meson_options.txt' file.
+     * @param[in] threads The number of threads - zero is treated as one.
+     * @return Result::Success when succeeded, otherwise the returned value depends on the problem type:
+     * Result::InsufficientCondition - an internal error possibly with memory allocation,
+     * Result::InvalidArguments - if unknown engine type chosen,
+     * Result::NonSupport - in case the engine type in the 'meson_options.txt' differs from the one specified in the @p engine argument,
+     * Result::Unknown - other. 
```

Comment:
Hi 
In doxygen syntax, you can use `@retval` to describe the return value.
ex)
```
     * @return Result::Success when succeeded, otherwise the returned value depends on the problem type:
     * @retval Result::InsufficientCondition An internal error possibly with memory allocation,
     * @retval Result::InvalidArguments If unknown engine type chosen,
     * @retval Result::NonSupport In case the engine type in the 'meson_options.txt' differs from the one specified in the @p engine argument,
     * @retval Result::Unknown other. 
```



before
![image](https://user-images.githubusercontent.com/7413838/116167197-b1ed6900-a73a-11eb-8478-dc1271484615.png)

after
![image](https://user-images.githubusercontent.com/7413838/116167269-d8ab9f80-a73a-11eb-825b-3e6ed0f2ba46.png)

Reply:
thx
Please check now. I used @retval in all cases where were more than 2 return values.
Also, in the case above, I used all ```@retval``` commands. I didn't mixed ```@return``` with ```@retval```.